content/news/2023/package-manager.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: "New Features in Pamac"
+title: "New Features In Pamac"
```

Comment:
https://capitalizemytitle.com/ says "in" should be lower case. So no need to change that again.